### DISCUSSION TITLE
求paddlepaddle-gpu与cuda\cudann的版本依赖

### DISCUSSION THREAD

**Discussion by MonsterSir** [PaddlePaddle/PaddleOCR#12390] [Q&A]
我没有找到paddlepaddle-gpu与cuda和cudann的相关版本依赖，是否有近几个版本的版本依赖推荐？

**Comment by ghplvh** [comment]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/14803974/6ee567d9-36d0-4ad0-83ed-c9ec8e51b091)
https://www.paddlepaddle.org.cn/documentation/docs/zh/install/Tables.html
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/14803974/4b5b12c9-d0a4-466b-bc5c-c29ef8f3793b)
https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/install/pip/linux-pip.html

**Comment by ghplvh** [comment]
他藏得好深，我看得时候也是翻了半天

**Comment by MonsterSir** [comment]
> ![image](https://private-user-images.githubusercontent.com/14803974/295890418-6ee567d9-36d0-4ad0-83ed-c9ec8e51b091.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ5ODMyOTgsIm5iZiI6MTcwNDk4Mjk5OCwicGF0aCI6Ii8xNDgwMzk3NC8yOTU4OTA0MTgtNmVlNTY3ZDktMzZkMC00YWQwLTgzZWQtYzllYzhlNTFiMDkxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTExVDE0MjMxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIxN2MyYWQxNDI4Yjg0OWEyNWU3MDM4MWViMmZhNGZjOTk2YWUzM2VjYmM2YzgyNmVhOGJhZmYyY2NkMjFmNWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Pll-_9moHqq6QWIiPtO5Ys4EaB06X9vpgl0nOPvxkFI) https://www.paddlepaddle.org.cn/documentation/docs/zh/install/Tables.html ![image](https://private-user-images.githubusercontent.com/14803974/295890237-4b5b12c9-d0a4-466b-bc5c-c29ef8f3793b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ5ODMyOTgsIm5iZiI6MTcwNDk4Mjk5OCwicGF0aCI6Ii8xNDgwMzk3NC8yOTU4OTAyMzctNGI1YjEyYzktZDBhNC00NjZiLWJjNWMtYzI5ZWY4ZjM3OTNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTExVDE0MjMxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMzOTlkMjc2OWQ2NDI5Y2Y2OTJiZDA1MjIxNzI5ZTE2MzY4YTgzZmNlYTRjMzJlMTlmOWU1YTlhMWEyYTQ2OWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4ASPMuZ2sLjJ7Lq0w5lRElEiunP800jzDrdplNyVnWo) https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/install/pip/linux-pip.html

### 非常感谢您的帮助，很有用！但我在安装paddleclas时，就会遇到faiss-cpu错误。
部分报错如下：
```
“note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for faiss-cpu
  Running setup.py clean for faiss-cpu
Failed to build faiss-cpu
ERROR: Could not build wheels for faiss-cpu, which is required to install pyproject.toml-based projects”
```
我尝试了paddlepaddle的2.5|2.6|2.7.0.1~2.7.0.3等版本，我的操作系统是linux，cuda==12.0，cudann=8，python=3.10，pip是最新的。非常奇怪的是我的faiss-cpu是已经安装好了的1.7.4，但是安装paddleclas还是会出现上述报错
